docs/roman/pipeline/calroman_level2.rst

```diff
@@ -0,0 +1,81 @@
+.. _calroman_level2:
+
+
+calroman_level2: Stage 2 Detector Processing
+==============================================
+
+:Class: `romancal.pipeline.Level2Pipeline`
+:Alias: calroman_level2
+
+The ``Level2Pipeline`` applies basic detector-level corrections to given exposure
+types (imaging, prism, and grism.). It is applied to one
+exposure at a time.
+It is sometimes referred to as "ramps-to-slopes" processing, because the input raw data
+are in the form of one or more ramps (integrations) containing accumulating
+counts from the non-destructive detector readouts and the output is a corrected
+countrate (slope) image.
+
+The list of steps applied by the ``Level2Pipeline`` pipeline is shown in the
+table below.
+
+
+========================================= ========= ========= =========
+ Step                                     WFI-Image WFI-Prism WFI-Grism
+========================================= ========= ========= =========
+ :ref:`dq_init <dq_init_step>`               x          x        x
+ :ref:`saturation <saturation_step>`         x          x        x
+ :ref:`dark_current <dark_current_step>`     x          x        x
+ :ref:`jump <jump_step>`                     x          x        x
+ :ref:`ramp_fitting <ramp_fitting_step>`     x          x        x
```

Comment:
`assign_wcs` and `flat` are missing

Reply:
Added flat and assign_wcs. Flat is now skipped for Prism & Grism data